crates/explorer/src/parse/bpf_upgradeable_loader.rs

```diff
@@ -112,6 +112,30 @@ pub fn parse_bpf_upgradeable_loader(
                 }),
             })
         }
+        UpgradeableLoaderInstruction::ExtendProgram { additional_bytes } => {
+            check_num_bpf_upgradeable_loader_accounts(&instruction.accounts, 4)?;
+            Ok(ParsedInstructionEnum {
+                instruction_type: "Close".to_string(),
```

Comment:
I believe here should be "ExtendProgram" instead of "Close"...

Reply:
Thanks for the notice! Fixed it